### PR TITLE
vultr: rename prefix vr_to vultr_

### LINKED ISSUE #42942: Rename the vultr module prefixes from vr_ to vultr_
##### SUMMARY

as pointed out by @dagwieers the prefix for vultr modules `vr_` is not recognizable and as vultr is a relatively short name, it would probably help in the future to free the vr_ "namespace" for others (with a much longer name". That is why we should rename all vultr modules prefixes to `vultr_`

Todos:
- [x] Check  BOTMETA
- [x] Modules
- [x] Docs in modules
- [x] Upgrade notes
- [x] Tests

##### ISSUE TYPE
 - Feature Idea

##### COMPONENT NAME

vultr

##### ANSIBLE VERSION

```
2.7
```

### PR DESCRIPTION
**Pull request by resmo**
##### SUMMARY
Free namespace vr_ for longer names than vultr. Fixes #42942 

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

vultr

##### ANSIBLE VERSION

```
2.7
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by resmo** [comment]
bot_status

**Comment by resmo** [comment]
ready_for_review

**Comment by resmo** [comment]
!needs_revision

File: lib/ansible/modules/cloud/vultr/vultr_dns_record.py [hunk 1]
```diff
@@ -39,10 +39,10 @@
     - CNAME
     - MX
     - SRV
-    - ALIAS
-    - SPF
+    - CAA
```
Comment (dagwieers) [inline lib/ansible/modules/cloud/vultr/vultr_dns_record.py:42]: Are these changes on purpose ?

**Review by dagwieers — APPROVED** [review]

File: lib/ansible/modules/cloud/vultr/vultr_dns_record.py (hunk 1 above)
Comment (resmo) [reply lib/ansible/modules/cloud/vultr/vultr_dns_record.py:42]: yes. fixes sanity tests.

**Comment by Spredzy** [comment]
shipit
